spec/requests/spree/api/v2/storefront/stripe/payment_intents_spec.rb

```diff
@@ -91,4 +94,174 @@
       expect(payment_intent.stripe_payment_method_id).to eq(stripe_payment_method_id)
     end
   end
+
+  describe 'payment_intents#confirm' do
+    subject do
+      post "/api/v2/storefront/stripe/payment_intents/#{payment_intent.id}/confirm", headers: headers
```

Comment:
@Vegann shouldn't this be patch as we're updating an existing resource/not creating a new one?